pkg/store/session_store.go

```diff
@@ -0,0 +1,78 @@
+package store
+
+import (
+	"context"
+	"database/sql"
+	"fmt"
+
+	"go.mau.fi/util/dbutil"
+
+	"go.mau.fi/mautrix-meta/messagix/cookies"
+	"go.mau.fi/mautrix-meta/messagix/types"
+)
+
+type MetaSession struct {
+	MetaID     int64
+	WADeviceID uint16
+	Cookies    *cookies.Cookies
+}
+
+func (s *MetaSession) Scan(row dbutil.Scannable) (*MetaSession, error) {
+	var metaID sql.NullInt64
+	var waDeviceID sql.NullInt32
+	var platform sql.NullString
+	var newCookies cookies.Cookies
+
+	err := row.Scan(
+		&metaID,
+		&platform,
+		&waDeviceID,
+		&dbutil.JSON{Data: &newCookies},
```

Comment:
How exactly does this automatic JSON storage work? Will it store the platform and, more importantly the push keys?